lib/tshield/request_vcr.rb

```diff
@@ -80,10 +80,20 @@ def method
 
     def save(raw_response)
       headers = {}
+
       raw_response.headers.each do |k, v|
+        next if k == 'set-cookie'
         headers[k] = v unless configuration.not_save_headers(domain).include? k
       end
 
+      field = raw_response.get_fields('Set-Cookie')
+
+      if !field.nil? && !field.empty?
+        cookies_values = []
+        field.each { |value| cookies_values.push(value) }
+        headers['Set-Cookie'] = cookies_values
+      end
+
```

Comment:
Can you extract the new logic to a new method please?
Removing code from here I think this issue can be solved.
https://app.sourcelevel.io/github/diegorubin/-/tshield/reviews/119/file/lib/tshield/request_vcr.rb#L81

Reply:
Hi Rubin, can u review the updates?